resqpy/model.py

```diff
@@ -1535,8 +1535,8 @@ def grid_connection_set_list(self):
       gcs_list = []
```

Comment:
For consistency should this be one of the "iter" methods, i.e. `model.iter_grid_connection_sets` ?

Reply:
Yes, good call, seeing as we have other breaking changes that will need attention in calling code, I'll change this to a generator.